### DISCUSSION TITLE
Regarding channelValue parameter for O3R cameras.

### DISCUSSION THREAD

**Discussion by Astronomia-maker** [ifm/ifm3d#401] [Q&A] [answered]
I am using O3R222 ( 4 pcs.) in my application. So sometime i have problem for crosstalk between these cameras. Currently i have set channelvalue= 2 for 2 cameras out of 4 and remaining have channelvalue = 4. 

For experiment,  i have set all cameras to different channelvalue (For e.g., 2,3,4,7) then i dont have any crosstalk issue so far.

So, is there any max limit to set channelvalue number or i can set any number?

**Comment by dekhanra** [comment, marked as answer]
Hi @Astronomia-maker, please refer to our documentation regarding the channel values in our website https://ifm3d.com/documentation/Technology/3D/AcquisitionParams/index_acquisition_params.html?highlight=channel%20value#channel-selection-and-channel-value
